general/development/tools/cibot.md

```diff
@@ -55,7 +55,7 @@ Please [file an issue](https://tracker.moodle.org/secure/CreateIssueDetails!init
 
 When you overrun the length limit many git tools do not display commits well. See how [github truncates ea6f548081](https://github.com/moodle/moodle/commits/ea6f5480818c31763f91a90a0cafb6a63ca18117) - it ruins the message and makes it harder for you to communicate your change to other developers.
 
-It is acknowledged that its often tricky to get a useful message in such a short space on the first line. However, the coding guidelines for git summary line length were established on the basis of [industry](https://github.com/blog/926-shiny-new-commit-styles) [best](http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/SubmittingPatches?id=aad7fb916a10f1065ad23de0c80a4a04bcba8437#n594) [practice](http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting). As the [Coding style](/general/development/policies/codingstyle#git-commits) mentions, do not be afraid to go into much more detail in your commit body.
+It is acknowledged that its often tricky to get a useful message in such a short space on the first line. However, the coding guidelines for git summary line length were established on the basis of [industry](https://github.com/blog/926-shiny-new-commit-styles) [best](http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/SubmittingPatches?id=aad7fb916a10f1065ad23de0c80a4a04bcba8437#n594) [practice](http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting). As the [Coding style](../policies/codingstyle/index.md#git-commits) mentions, do not be afraid to go into much more detail in your commit body.
```

Comment:
There is one more reference to /general/development/policies/codingstyle in line 88.